### PR TITLE
2773 tags overflow

### LINKED ISSUE #2773: Tags overlap footer in map card
captura de pantalla 2015-03-14 a las 11 36 23

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Fixes #2773. Visit the issue to see how tags look.

@xavijam PTAL. Thanks!

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2501/))

File: app/assets/stylesheets/new_common/map-card.css.scss [hunk 1]
```diff
@@ -61,18 +61,38 @@ $opacityTransition: 250ms;
   text-align: center;
   margin-bottom: $sMargin-element;
 }
+.MapCard-tags {
+  height: 1 * $sLineHeight-normal;
+  display: -webkit-box;
```
Comment (xavijam) [inline app/assets/stylesheets/new_common/map-card.css.scss]: what about non webkit browsers? :P

File: app/assets/stylesheets/new_common/map-card.css.scss [hunk 2]
```diff
@@ -61,18 +61,38 @@ $opacityTransition: 250ms;
...
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
```
Comment (xavijam) [inline app/assets/stylesheets/new_common/map-card.css.scss]: Maybe we should create a local mixin with that behaviour.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2529/))

**Comment by alonsogarciapablo** [comment]
@xavijam I've created [a mixin with the duplicated css](https://github.com/CartoDB/cartodb/commit/88e528fecdd822e463befccc60383befdad8e97e).

All good now?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2532/))

**Comment by xavijam** [comment]
Looks good!

**Comment by saleiva** [comment]
bump!

**Pull request merged**